lib/active_merchant/billing/gateways/cyber_source.rb

```diff
@@ -807,26 +807,35 @@ def network_tokenization?(payment_method)
         payment_method.is_a?(NetworkTokenizationCreditCard)
       end
 
+      def subsequent_nt_apple_pay_auth(source, options)
+        return unless options[:stored_credential] || options[:stored_credential_overrides]
+        return unless @@payment_solution[source]
```

Comment:
This change was reverted previously because Network Tokens were falling down the code path to update the `commerce_indicator` to `internet` causing an increase in `General Decline`. This change now verifies the payment method is AP or GP before updating the `commerce_indicator`